scripts/add_cluster.sh

```
@@ -0,0 +1,154 @@
+fi
+
+# Enable error handling
+set -euo pipefail
+trap 'echo "Error on line $LINENO: $(tail -n +$LINENO "$0" | head -n 1)"; exit 1' ERR
+trap "echo 'Script interrupted by user'; exit" SIGINT
+
+# Main menu
+PLATFORM=$(gum choose "GCP" "AWS")
+
+# Environment variables
+export ARGOCD_SERVER=$(gum input --placeholder "Enter ArgoCD Server")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)